tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/_init_deploy.tpl

```diff
@@ -16,7 +16,7 @@
           -n $groupName \
           -f /mnt/testresources/test-resources.json \
           --parameters /mnt/testresources/parameters.json \
-          --parameters testApplicationOid=$AZURE_CLIENT_OID &&
+          --parameters testApplicationOid=$AZURE_CLIENT_OID > /dev/null &&
```

Comment:
I understand this will help with leaks but what about diagnosing issues? Do we have enough information in the logs to figure that out? I guess it might not matter too much once we move to using the common deployment scripts with your other workitem.

Reply:
I will miss the logging here, but errors printed to stderr will still get logged. I figure in the interim until the common script move, any non-transient issues with which we need stdout to diagnose are easy enough to investigate on an individual basis.